groups/createElements.js

```diff
@@ -123,28 +123,40 @@ const createGroupCreationModal = (onClose = () => {}, onSubmit = () => {}) => {
                 </button>
               </div>
             </div>
+            <div class="input" id="description">
+              <label for="description" class="input__label">Description</label>
+              <div class="input__container">
+                <textarea
+                  class="input__field"
+                  name="description"
+                  placeholder="This group is for..." 
+                ></textarea>
+              </div>
+            </div>
```

Comment:
all new UI components should meet accessibility standards. Use ARIA labels where necessary to ensure keyboard navigation works as expected.